app/src/free/java/com/kirylshreyter/diamondapp/factories/UrlsFactory.java

```diff
@@ -0,0 +1,15 @@
+package com.kirylshreyter.diamondapp.factories;
+
+import com.kirylshreyter.diamondapp.BuildConfig;
+
+public class UrlsFactory {
```

Comment:
You have 2 absolutely same classes UrlsFactory.
Move it to main folder and remove duplicate